Gemfile.lock

```diff
@@ -1,25 +1,35 @@
 PATH
   remote: .
   specs:
-    twilio_mock (0.4.6)
+    twilio_mock (0.4.7)
+      activesupport (>= 3)
       twilio-ruby (~> 5.3, >= 5.3.1)
       webmock (~> 3.0, >= 2)
 
 GEM
   remote: https://rubygems.org/
   specs:
+    activesupport (5.2.1)
```

Comment:
can you avoid adding this dependency? check [my comment about this](https://github.com/MaicolBen/twilio_mock/pull/13#discussion_r208054218)